src/seer_utils.erl

```diff
@@ -0,0 +1,54 @@
+-module(seer_utils).
+
+-include("seer.hrl").
+
+-export([carbon_format/3]).
+
+-spec carbon_format(binary(), binary(), [read_metric()]) -> carbon_batch().
+carbon_format(Prefix, Host, Metrics) ->
+    Timestamp = erlang:system_time(second),
+    lists:flatten([carbon_string(Prefix, Host, Metric, Timestamp) || Metric <- Metrics]).
```

Comment:
Any reason why you don't use `iolist_to_binary`?

Reply:
Because the `carbon_send_batch/2` function expects a list of carbon-formatted metrics and sends them one by one. `iolist_to_binary` would instead return a single big binary.

Reply:
Ok :)